src/utils/queryBuilder.ts

```diff
@@ -0,0 +1,19 @@
+import composeQuery from './composeQuery';
+
+interface VariableToKeyItem {
+  key: string;
+  value: string[] | string | number | boolean;
+}
+
+const queryBuilder = (items: VariableToKeyItem[]): string => {
+  return items.reduce((query, item) => {
+    if (Array.isArray(item.value) && item.value.length) {
+      return composeQuery(query, item.key, item.value.join(','));
+    } else if (item.value && !Array.isArray(item.value)) {
```

Comment:
simplified this, no need for 4 if statements

Reply:
Does this work with edge cases (number = 0 or boolean = false)? For example https://api.hel.fi/linkedevents/v1/event/?is_free=false and https://api.hel.fi/linkedevents/v1/event gives different results.

Reply:
Yeah actually noticed this today and fixed it in my current branch